src/abilities/Magma-Spawn.js

```diff
@@ -315,7 +315,7 @@ export default G => {
 				}
 
 				// Movement
-				var hurl = function(_path) {
+				let hurl = _path => {
 					let target = arrayUtils.last(_path).creature;
```

Comment:
Random leading underscore, not sure there is a pattern in our code base that would encourage this?

Reply:
there's many usages of underscores, especially in the Abilities code